process/track/baseBlockTrack.go

```diff
@@ -51,6 +51,8 @@ type baseBlockTrack struct {
 	mutHeaders                  sync.RWMutex
 	headers                     map[uint32]map[uint64][]*HeaderInfo
 	maxNumHeadersToKeepPerShard int
+	ComputeLongestSelfChainFunc func() (data.HeaderHandler, []byte, []data.HeaderHandler, [][]byte)
```

Comment:
Let's not use these exported function pointers. Reason: I can take the component, set this function pointer to nil and then the component will break in panic

Reply:
Removed, as it is used only from outside of the component